ajax/libs/folktale/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "folktale",
+  "filename": "umd/folktale.min.js",
+  "version": "2.0.1",
+  "description": "A suite of libraries for generic functional programming in JavaScript that allows you to write elegant modular applications with fewer bugs and more reuse.",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/origamitower/folktale"
+  },
+  "keywords": [
+    "functional",
+    "generic",
+    "fantasy-land",
+    "folktale",
+    "monads"
+  ],
+  "author": "Quildreen Motta",
+  "license": "MIT",
+  "homepage": "http://folktale.origamitower.com/",
+  "npmName": "folktale",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
```

Comment:
May be better to takes files from dist/umd and decrease path lenght?

Reply:
If the author release other files which can be used in front-end that under `dist` but not under `umd`, then we need to update the auto-config.
It's not a good choice, because we know that we use the excluded pattern in the `dist`, but if we use `dist/umd` in the basePath, it's included pattern.

Thanks.

Reply:
I prefer to use `dist/umd`